script.js

```diff
@@ -0,0 +1,47 @@
+function retrieveBooks() {
+  const books = JSON.parse(localStorage.getItem('Books'));
+  document.getElementById('books-list').innerHTML = '';
+  for (let i = 0; i < books.length; i += 1) {
+    document.getElementById('books-list').innerHTML
+          += `<p>${books[i].title}</p>
+              <p>${books[i].author}</p>
+              <button type="button" onclick="removeBook(${i})">Remove</button>
+              <hr>
+              `;
+  }
+}
+
+function createBook(e) {
+  e.preventDefault();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+
+  const book = {
+    title,
+    author,
+  };
+
+  if (localStorage.getItem('Books') === null) {
+    const books = [];
+    books.push(book);
+    localStorage.setItem('Books', JSON.stringify(books));
+  } else {
+    const books = JSON.parse(localStorage.getItem('Books'));
+    books.push(book);
+    localStorage.setItem('Books', JSON.stringify(books));
+  }
+  retrieveBooks();
+  document.getElementById('form').reset();
+}
+
+document.getElementById('form').addEventListener('submit', createBook);
+
+retrieveBooks();
+function removeBook(i) {
+  const books = JSON.parse(localStorage.getItem('Books'));
+  books.splice(i, 1);
+  localStorage.setItem('Books', JSON.stringify(books));
+  retrieveBooks();
+}
+
+removeBook(1000000);
```

Comment:
- [ ] I don't really understand the purpose of this function being called with this parameter. May this be a typo?

Reply:
@Lordkaito Thank you for your kindly response. It's to avoid linter error because it stated the function was declared but never used, but I actually use it inside the onclick event handler of the remove button and I only found this solution. Any advice?

Reply:
That's probably because you used innerHtml to declare the use of your `removeBook` function. You can try something like this instead:
```
const removeButton = document.getElementById('remove-button')
removeButton.addEventListener('click', removeBook)
```
this way, you are declaring the event on your JavaScript file, which should be better and should avoid the linter error. Also, remember to use an id within your button and to use that id when you create your `removeButton` variable.
If that doesn't work, you can always try to disable the linter check for a certain line, but this is not usually recommended.

Reply:
@Lordkaito Thank you so much for your help. We will try that.